server/profile.go

```diff
@@ -91,8 +91,87 @@ func GetProfile(w http.ResponseWriter, r *http.Request) {
 }
 
 func EditProfile(w http.ResponseWriter, r *http.Request) {
+	//Connect to datastore
+	ctx := context.Background()
```

Comment:
Context needs to come from the request to allow cancellation, etc: https://golang.org/pkg/net/http/#Request.Context